Docs/source/install/hpc/cori.rst

```diff
@@ -37,10 +38,16 @@ We use the following modules and environments on the system (``$HOME/knl_warpx.p
    module swap PrgEnv-intel PrgEnv-gnu
    module load cmake/3.20.5
    module load cray-hdf5-parallel/1.10.5.2
-   module load cray-fftw
+   module load cray-fftw/3.3.8.4
    module load cray-python/3.7.3.2
 
-   export CMAKE_PREFIX_PATH=$PWD/adios2-2.7.1-knl-install:$CMAKE_PREFIX_PATH
+   export PKG_CONFIG_PATH=$FFTW_DIR/pkgconfig:$PKG_CONFIG_PATH
+   export CMAKE_PREFIX_PATH=$HOME/sw/adios2-2.7.1-knl-install:$CMAKE_PREFIX_PATH
+
+   if [ -d "$HOME/sw/venvs/knl_warpx" ]
+   then
+     source $HOME/sw/venvs/knl_warpx/bin/activate
```

Comment:
This command might be confusing since the commands to setup the virtual environment are described after this line. Maybe add a comment like "This assumes that the knl_warpx virtual environment has been setup (see below).".

Reply:
It's intentionally guarded: nothing will happen if the directory (`-d`) does not yet exist.
That way, one can follow the instructions top-to-bottom in the page :)